logic/atrium-logic-jvm/src/main/kotlin/ch/tutteli/atrium/logic/creating/filesystem/hints/hints.kt

```diff
@@ -45,6 +48,18 @@ fun Descriptive.DescriptionOption<Descriptive.FinalStep>.withFileAttributesFailu
         }
     }
 
+fun Descriptive.DescriptionOption<Descriptive.FinalStep>.withPathAttributesFailureHint(
+    provider: SubjectProvider<Path>
+): Descriptive.DescriptionOption<DescriptiveAssertionWithFailureHint.FinalStep> =
+    withFailureHintBasedOnDefinedSubject(provider) { path ->
+        explainForResolvedLink(path) {
+            describeWas(it.type)
+        }
+    }
+
+private val Path.type: DescriptionPathAssertion
+    get() = if (isAbsolute) ABSOLUTE_PATH else RELATIVE_PATH
```

Comment:
I don’t think `Path.type` is a very descriptive name for this. However, I also have nothing better to offer. Maybe `Path.relativness` or `Path.declarationType`? I just wanted to comment it, I’ll leave it to you (or @robstoll) to decide what you like best.

Reply:
Should be removed, this should no longer be used after simplifying isRelative in Default... 
I think it would be easiest to revert all changes in Default... and copy startsWith and adapt to isAbsolute

Reply:
Moved everything to Default...